languages/swift/exercises/concept/tuples/.docs/after.md

```diff
@@ -0,0 +1,84 @@
+[Tuples][tuples] are used to combine multiple values into a single compound value where each of the values may have a different type. The compound value has a type that is the combination of the individual values' types.
+
+```swift
+let order: (String, Double, Int) = ("biscuits", 2.99, 3)
+```
+
+The fact that tuples are single values allows them to be used to [return multiple values from a function][multiple-return-values]:
+
+```swift
+func longestAndShortest(_ strings: [String]) -> (longest: String?, shortest: String?) {
+  var longestLength = Int.min, shortestLength = Int.max
+  var longAndShort: (longest: String?, shortest: String?) = (longest: nil, shortest: nil)
```

Comment:
I'm thinking about that implementation. What about using two separate strings `longest` and `shortest` and to construct the tuple only in return statement? That might also underline the idea of using tuples to pass multiple return values (while within the function and outside it can be worked with the separate variables).